lib/avro_turf/test/fake_prefixed_confluent_schema_registry_server.rb

```diff
@@ -64,7 +64,8 @@ class FakePrefixedConfluentSchemaRegistryServer < FakeConfluentSchemaRegistrySer
 
     # Note: this does not actually handle the same schema registered under
     # multiple subjects
-    schema_id = SCHEMAS.index(schema)
+    context, _subject = parse_qualified_subject(params[:subject])
+    schema_id = SCHEMAS[context].index(schema)
```

Comment:
This was already fixed in [the parent](https://github.com/dasch/avro_turf/blob/b0cb5f19c76cd1e94729f25dd673e64655ecbafe/lib/avro_turf/test/fake_confluent_schema_registry_server.rb#L121-L124) but was failing here so all I did was copy the same behavior